cedar-policy/CHANGELOG.md

```diff
@@ -423,7 +431,8 @@ Cedar Language Version: 2.0.0
 Cedar Language Version: 2.0.0
 - Initial release of `cedar-policy`.
 
-[Unreleased]: https://github.com/cedar-policy/cedar/compare/v3.1.3...main
+[Unreleased]: https://github.com/cedar-policy/cedar/compare/v3.2.0...main
+[3.2.0]: https://github.com/cedar-policy/cedar/compare/v3.1.3...v3.2.0
```

Comment:
Also done in #851

Reply:
#851 is towards `main` and this is towards `release/3.2.x`.  Ideally we'd have #851's changes on `release/3.2.x` before we merge this, and I'll rebase.

Reply:
#853 brings #851's changes to `release/3.2.x`